prime-angular-frontend/src/app/modules/adjudication/shared/components/enrollee-table/enrollee-table.component.html

```diff
@@ -15,16 +15,23 @@
           <td mat-cell
               *matCellDef="let row;">
             <div class="d-flex flex-wrap triage-container">
+
               <app-triage *ngIf="row.currentStatusCode === EnrolmentStatus.UNDER_REVIEW"
                           class="align-self-start"
                           [enrolleeId]="row.id"
                           [assigned]="(row.adjudicatorIdir)"
                           (reload)="onReload()"></app-triage>
 
+              <app-contextual-enrolment-confirmation *ngIf="row.requiresConfirmation"
```

Comment:
Im assuming this might be a corner case, but if a user submits and is automatically adjudicated. Then submits again and is under review. Is there a case where requiresConfirmation && EnrolmentStatus = UnderReview?

if so maybe just add "&& row.currentStatusCode !== EnrolmentStatus.UNDER_REVIEW" to this condition so we can cover that case and having to many objects here.

Reply:
No this shouldnt be possible because you can only get requiresConfirmation if your previous status was RequiresTOA, so if you were Under Review your last status would be editable